htdocs/admin/ticket.php

```diff
@@ -494,11 +549,92 @@
 print '</td>';
 print '</tr>';
 
-print '</table><br>';
+// Auto create intervention linked to ticket
+print '<tr class="oddeven"><td>'.$langs->trans("TicketsAutoCreateIntervention").'</td>';
+print '<td class="left">';
+if ($conf->use_javascript_ajax) {
+	print ajax_constantonoff('TICKET_AUTO_CREATE_FICHINTER_CREATE');
```

Comment:
Is it a process to create an intervention (module intervention) from a ticket (module ticket).
If yes, because this is an intermodule-workflow, option must be moved into the module "Workflow".
To keep things clear, in Dolibarr, there is no "automatic" interaction on business event between different modules, except the one defined into module Workflow.

Reply:
This is an automatic action that was already in Ticket module. TICKET_AUTO_ASSIGN_CONTRACT_CREATE was already part of it.

I moved both to Workflow module, and enhanced auto assign of contracts to optionally use parent companies and link only contracts with ticket support service.